python/src/usecases/linthurst_dataset.py

```diff
@@ -0,0 +1,79 @@
+"""
+Use case : Linthurst data set
+=============================
+"""
+import openturns as ot
+
+
+class LinthurstDataset:
```

Comment:
maybe the module & class names could be simplified:
"linthurst_dataset.LinthurstDataset().data" -> "linthurst.Linthurst.data"
like we have for "coles.Coles.wavesurge"